test/integration/apiserver/openapi/openapiv3_test.go

```diff
@@ -46,11 +46,10 @@ import (
 
 func TestOpenAPIV3SpecRoundTrip(t *testing.T) {
 	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, genericfeatures.OpenAPIV3, true)()
-	controlPlaneConfig := framework.NewIntegrationTestControlPlaneConfigWithOptions(&framework.ControlPlaneConfigOptions{})
-	controlPlaneConfig.GenericConfig.OpenAPIConfig = framework.DefaultOpenAPIConfig()
-	controlPlaneConfig.GenericConfig.OpenAPIV3Config = framework.DefaultOpenAPIV3Config()
```

Comment:
don't you need to pass this to the StartTestServer() function 
```
				ModifyServerConfig: func(config *controlplane.Config) {
					config.GenericConfig.OpenAPIConfig = framework.DefaultOpenAPIConfig()
```

Reply:
You don't - that's a default...